### DISCUSSION TITLE
HTML examples are not working

### DISCUSSION THREAD

**Discussion by abnair01** [patternfly/patternfly#5661] [Q&A]
React examples are working fine. I can interact and see the behavior. But when I click on HTML, the components don't respond. Am I missing something? Check out the video.

https://github.com/patternfly/patternfly/assets/112004869/66139589-50c9-452c-9fff-60bb2d44724a

**Comment by hdnarola** [comment]
Facing same issue here, 
It would be better if it's been described clearly or do we need to use only react version to have interactive components?

**Comment by mcarrano** [comment]
@abnair01 @hdnarola you are correct that the HTML examples are not interactive. The purpose of offering an HTML version of the library is for use by teams who may want to add their own Javascript to our components. React is our Javascript framework of choice and as you noted you can see interactive examples there.

**Reply by grafuls** [reply]
Is there a way to add the Patternfly js libraries to an HTML version of it?
I started working on a [`Flask-Patternfly`](https://github.com/grafuls/flask-patternfly) extension basing all the content on the HTML examples and would be missing the js functionality and don't want to have to deal with react inside of it.

**Reply by nicolethoen** [reply]
The only JS libraries the PatternFly team officially supports to fully implement interactive components for PatternFly 4 & PatternFly 5 is our React implementation.  There are a few other community projects which exist to provide alternative JS implementations, such as the project listed on [this page](https://www.patternfly.org/get-started/community) under 'Creating new communities'.

I am not aware of a current implementation or community project implementing the components using vanilla JavaScript at this time.
